### DISCUSSION TITLE
Plausible Analytics - Track main domain and subdomain together

### DISCUSSION THREAD

**Discussion by Kristian321** [plausible/analytics#1952] [Self-Hosted Support]
Hello there

I have an issue with subdomain tracking in Plausible Analytics, that I hope somebody can assist with.

In my analytics report the subdomain in reported as a source for traffic to the main domain. However, I would like an aggregated overview of both sub- and main domain without self referrals. 

I am not sure how/if this is achievable.

The same tracking script is implemented on the main and subdomain and looks like this:

<script defer data-domain="mydomain.dk,mysubdomain.dk" src="https://plausible.mydomain.dk/js/plausible.js"></script>

It does not solve the issue since I got traffic to mydomain.dk where the source was mysubdomain.dk yesterday...

Does anybody know how to solve this issue... is there even a solution?

I will appreciate any insights.

Thanks in advance

Kristian

**Comment by metmarkosaric** [comment]
hi @Kristian321! have you tried roll up reporting? https://plausible.io/docs/plausible-script#is-there-a-roll-up-view

**Reply by marcbria** [reply]
That's perfect.

Case use:
I manage a service with 50 webapps (different domains) that now got it's own plausible analytics.
With the rollup reporting now I can have a global view of the whole service... that is not useful for anybody than me, but it makes me VERY happy. ;-)

Thanks a lot for plausible is a great tool.

**Comment by Kristian321** [comment]
Hi Mako

Thank you for the link!


Mange hilsener/Best regards

*Kristian Kvistgaard*
Digital Marketing Konsulent



*Tilmeld dig vores nyhedsbrev her <https://www.opvaekst.dk/#Nyhedsbrev>*
Website   www.opvaekst.dk
<https://www.opvaekst.dk/?utm_source=signature&utm_medium=email>
Telefon    71143619
Adresse   Flæsketorvet 28, 1711 København V


Den tor. 9. jun. 2022 kl. 17.20 skrev Marko Saric ***@***.***

**Comment by Plaidmustache** [comment]
I am having the same issue, but i am using wordpress, with two separate WP installs on a main domain, and subdomain. Is this achievable with the wordpress plugin installed on both sites, or do i HAVE to do the manual install of the script with these changes in order to track properly across the mainsite + subdomain.  thanks, i cannot find any info about this usecase online.
